log/log_test.go

```diff
@@ -1 +1,160 @@
 package log
+
+import (
+	"bytes"
+	"log"
+	"strings"
+	"testing"
+)
+
+func TestOutputf(t *testing.T) {
+	const string1 = "asdf123"
+	buf := new(bytes.Buffer)
+
+	log.SetOutput(buf)
+	outputf(LevelDebug, string1, nil)
+
+	// test 1: outputf correctly prints string
```

Comment:
if you planning to add more test, please do. If not, remove "test 1" here and at other places.